database/java/2018/2372.yaml

```diff
@@ -0,0 +1,35 @@
+---
+cve: 2018-2372
+title: CVE in org.commonjava.ssl:ssl-tools
+description: >
+    A plain keystore password is written to a system log file in SAP HANA Extended Application Services, 1.0, which could endanger confidentiality of SSL communication.
+cvss_v2: 4.0
+references:
+    - https://blogs.sap.com/2018/02/13/sap-security-patch-day-february-2018/
+    - https://launchpad.support.sap.com/#/notes/2589129
+
+affected:
+    - groupId: org.commonjava.ssl
+      artifactId: ssl-tools
+      version:
+        - "<=1.1"
+      fixedin:
+        - "!FIXME!"
```

Comment:
This is a relatively new vulnerability identified in Feb 2018 itself, Fix is not yet released as far as I can see.